### DISCUSSION TITLE
Set client to receiving only

### DISCUSSION THREAD

**Discussion by FlamingFlamingo42** [rustdesk/rustdesk#1370] [General]
Is there a way to set the client to only receive remote help?

So an admin could connect to the pc of some office user and help them, but the office user wont be able to connect to any other pcs/remote control them?

**Comment by celevra** [comment]
+1

UI with connect options, could be too much for a normal user.
The normal user just needs a little Window with the ID, even without the password, it is enough to click "allow" on a connection request


would be also cool if there is an api to retrieve die Client ID
in Teamviewer you can read the client id from the registry

we have a little helper client installed, if someone presses ctrl+alt+z teamviewer is started and the client id is send to our rmm were we see a sign ala "client 'a' needs help clientid: ...."
